tor_ocr/main.py

```diff
@@ -36,6 +38,15 @@
 
     u_tor_post_id.reply(ocr_magic)
 """
+dotenv.load_dotenv()
+
+b_api = BlossomAPI(
+    email=os.environ.get('TOR_OCR_EMAIL'), 
+    password=os.environ.get('TOR_OCR_PASSWORD'), 
+    api_key='TOR_OCR_BLOSSOM_API_KEY', 
```

Comment:
Is the API key supposed to be obtained from the os environment variables?

Reply:
Yes, this was implemented due to direction from @itsthejoker

Reply:
(He's mentioning that that's a bare string, not pulling an environment variable 😉 )

Reply:
Woops, nice catch!

Reply:
Should be fixed now.